Common/Inc/Interchip.h

```diff
@@ -2,15 +2,16 @@
 * Interchip packets
 * Packets should be a multiple of 16 bits
 */
+#include "Common.h"
 
 typedef struct
 {
-  int16_t PWM[12];
+  int16_t PWM[PWM_NUM_CHANNELS];
```

Comment:
No guarantee that we have the same number of PPM inputs as we do outputs (. However, this does make the packets the same size. No harm in making the last 4 data 0.